src/index.html

```diff
@@ -24,11 +26,8 @@
     <link rel="icon" href="favicon.png" type="image/png" />
     <link rel="icon" href="favicon.svg" type="image/svg+xml" />
     <link
-      rel="preload"
-      href="./Inter.var.woff2?v=3.19"
-      as="font"
-      type="font/woff2"
-      crossorigin
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap"
```

Comment:
We actually only need 100–800, though not sure this makes any difference. And also don't need any of the font-style (oblqiue) stuff right now.